src/main/java/org/zalando/nakadi/repository/db/StorageDbRepository.java

```diff
@@ -0,0 +1,74 @@
+package org.zalando.nakadi.repository.db;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Optional;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+import org.zalando.nakadi.annotations.DB;
+import org.zalando.nakadi.domain.Storage;
+
+@DB
+@Repository
+public class StorageDbRepository extends AbstractDbRepository {
```

Comment:
I'm not sure about you policy regarding uppercase for SQL reserved words. It seems that you follow it but not in all places.